content/md/assets/family-by-family-pam-migration.md

```diff
@@ -0,0 +1,104 @@
+
+## _Step 3_ | Create the API credentials
+Then, the API will be used to create the asset family and your assets in the new Asset Manager.
+To create the API credentials in the PIM :
+1. Run `create-api-credentials`
+2. And store the API credentials
+
+## _Step 4_ | Import the PAM assets
+To import the PAM assets for each asset family, you can use our dedicated tool CSVToAsset with the CSV files for this asset family and the API credentials.
+In the CSVToAsset tool, for each asset family:
+1. Run `make migration` with the chosen asset family code for your family in the new Asset Manager
+2. Run `create asset family`
+3. Run `merge 2 CSV files in 1`
+4. Run `import assets into the PIM through API`
+
+## _Step 5_ | Migrate the PAM asset collection product attributes
+The PAM asset collection product attributes will be linked to one of the family created.
+In the PIM, for each PAM asset collection product attribute to migrate:
+- Run `migrate-pam-attributes` command with the chosen asset family code and the existing attribute code
+
+## _Step 6_ | Check and test the migration
+Yeah, well done, your assets are successfully migrated to the new Asset Manager. But we strongly recommend you to make some checks and tests to verify that everything is ok and discover more our new Asset Manager.
+
+To view all your assets:
+1. Go to the `Assets` menu
+1. Your asset family will be selected by default and all your assets will automatically appear in the grid
+1. You can search for an asset or display an asset by clicking on it
+If you want to go further, please read this article [Create and display assets](create-and-display-assets.html).
+
+To view how are structured your asset families:
+1. Go the `Assets` menu`
+1. Choose an asset family on the left
+1. Click on the button `Edit family` at the top right of the grid
+1. View the family attributes in `Attributes` tab, you can translate the attributes labels for each locale
+1. View the family properties in `Properties` tab, you can translate the family labels for each locale
+
+If you want to go further, please read this article [Manage asset families](manage-asset-families.html)
+
+To view the assets linked to a product:
+1. Click on a product on the product grid
+1. Click on the `Assets` tab, all the assets linked to this product will be displayed
+If you want to go further, please read this article [Work with asset in an asset collection atribute](work-with-assets-in-an-asset-collection-attribute-ee-only.html).
+
+## _Step 6_ | Set transformations (optional)
+In the new Asset Manager, transformations are no more mandatory. So if you used before fake transformations (100% or 99%), you don't need anymore to set transformations.
+
+If you were using some real transformations in the PAM, the variations of the assets are migrated during the step 3 but the transformations settings are not migrated.
```

Comment:
```suggestion
If you were using some real transformations in the PAM (for example, a `thumbnail` transformation), the variations of the assets will be migrated during the step 3 but the transformations settings won't be migrated since we totally change the format of the transformations settings.
```